src/components/Posts.js

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from 'react';
+import styles from './Posts.module.css';
+
+function Posts({link}) {
+    useEffect(() => {
+        const script = document.createElement('script');
+        script.src = "https://medium-widget.pixelpoint.io/widget.js";
+        script.async = true;
+        script.onload = () => {
+            MediumWidget.Init({
+                renderTo: `.${styles.mediumWidgetPosts}`,
+                params: {
```

Comment:
Can we filter this to only include the ones tagged with "Open Source"? I've asked the tech blog team to add this to their list of recommended tags depending on the article type. This would make the articles listed more relevant here but the downside would be fewer articles displayed overall...

Reply:
@massdosage I was thinking about that and I'm in 2 minds. We have very few blogposts related to Open Source. We should encourage teams to write more often :)

Reply:
Unfortunately the Pixel Point widget doesn't seem to apply any filtering, I tried it with https://medium.com/expedia-group-tech/tagged/open-source but it shows the same posts as https://medium.com/expedia-group-tech 😞

Reply:
OK, so let's go without filtering for now, we can discuss if it's worth trying to add it back in later.